libs/sdk-ui-tests-e2e/cypress/tools/chart.ts

```diff
@@ -154,4 +156,19 @@ export class Chart {
             });
         return cy.wrap(result);
     }
+
+    hasTooltipContents(index: number, tooltip: string[]) {
+        this.getTooltipContents(index).should("deep.equal", tooltip);
+    }
+
+    clickFirstCell() {
```

Comment:
this function should receive param is index, so that we can pass any index to this function, not only the first one